modules/core/pom.xml

```diff
@@ -68,6 +68,12 @@
             <version>${javax.cache.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
We should consider adding JNA, but the decision may depend on our ideas about Unsafe-less client/server implementation. For now, I'm concerned about adding JNA to core dependencies list.

I would suggest allocating memory via plugin, e.g. direct-io